frontend/src/features/leases/detail/LeasePages/tenant/Tenant.tsx

```diff
@@ -55,10 +55,12 @@ export const Tenant: React.FunctionComponent<ITenantProps> = ({ nameSpace }) =>
               (tenantNote: string, index) =>
                 !!tenantNote && (
                   <Styled.SpacedInlineListItem key={`notes-${index}`}>
-                    <TenantNotes
-                      disabled={true}
-                      nameSpace={withNameSpace(nameSpace, `tenantNotes.${index}`)}
-                    ></TenantNotes>
+                    <FormSection>
```

Comment:
@jdtoombs isn't notes already part of a formsection? is this necessary?

Reply:
This keeps it consistent with the other sections (contact info, organization contact info) as well as provides the appropriate width and padding